pkg/transformer/openshift/openshift.go

```diff
@@ -135,6 +135,24 @@ func getAbsBuildContext(context string, composeFileDir string) (string, error) {
 		return "", err
 	}
 	prefix := strings.Trim(string(out), "\n")
+	index := strings.Index(context, prefix)
+	if index != -1 {
+		if prefix == "" {
+			composeFilesplit := strings.Split(composeFileDir, "/")
+			contextsplit := strings.Split(context, "/")
+			var i int
```

Comment:
no need to state `var i int`, you can do this in the range: `for i := range contextsplit`